### PR TITLE
blade-template

### PR DESCRIPTION
**Pull request by BuiMinhDung711**

### CODE REVIEW

**Comment by BuiMinhDung711** [comment]
ready

File: app/Http/Controllers/UserController.php [hunk 1]
```diff
@@ -36,23 +39,30 @@ public function store(Request $request)
      */
     public function show(User $user)
     {
-        //
+        $projects = $user->projects;
+        return view('users.show', [
+            'user' => $user,
+            'projects' => $projects,
+        ]);
```
Comment (ghost) [inline app/Http/Controllers/UserController.php]:
```suggestion
        return view('users.show', compact('user', 'projects'));
```

**Comment by BuiMinhDung711** [comment]
ready

**Review by ghost — APPROVED** [review]
merged

**Comment by ghost** [comment]
merged

**Pull request merged**